streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1288,6 +1326,10 @@ int currentNumIterations() {
         return numIterations;
     }
 
+    long bufferSize() {
```

Comment:
This function seems not used.

BTW if we do not maintain the local `bufferSize` then we would not need it anyways :)